ea-domain/src/main/java/com/nbe2/domain/bookmark/BookmarkRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.nbe2.domain.bookmark;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface BookmarkRepository extends JpaRepository<Bookmark, Long> {
+
+    List<Bookmark> findByUserId(Long userId);
+
+    Optional<Bookmark> findByEmergencyRoomId(Long emergencyRoomId);
```

Comment:
어제 말씀하신대로 한 응급실을 즐겨찾기한 사용자가 여러 명일 수 있어서,
여기서 조건을 EmergencyRoomId만 걸고 조회하면 첫 번째로 조회된 Bookmark가 찾으려는 userId의 데이터가 아닐 수 있어요!

그래서 아래처럼 where 조건을 EmergencyRoomId랑 UserId 같이 AND로 걸어야 의도하신 대로 데이터가 조회될 거 같습니당
```suggestion
    Optional<Bookmark> findByEmergencyRoomIdAndUserId(Long emergencyRoomId, Long userId);
```